logging/zap/options_test.go

```diff
@@ -0,0 +1,15 @@
+package grpc_zap
+
+import (
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zapcore"
+	"math"
+	"testing"
+	"time"
+)
+
+func TestDurationToTimeMillisField(t *testing.T) {
+	val := DurationToTimeMillisField(time.Microsecond * 100)
+	assert.Equal(t, val.Type, zapcore.Float32Type, "should be a float type")
+	assert.Equal(t, math.Float32frombits(uint32(val.Integer)), float32(0.1), "sub millisecond values should be correct")
```

Comment:
`math.Float32frombits(uint32(` feels like it's relying on knowledge of zap internals, is there any other way we can get the processed value?

Reply:
This was a tricky one to figure out, wasnt sure of it myself tbh but couldn't think of a way to do it without knowing how zapFlags worked.
If we had #114 merged we could probably grab it off the log line in an interceptor test and assert its not 0 therefore not testing this method directly.